docker/Dockerfile

```diff
@@ -103,6 +103,9 @@ RUN echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontain
     wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_20.04/Release.key -O- | apt-key add -
 RUN apt-get update && apt-get -y install skopeo && ln -s /usr/bin/skopeo /sbin/skopeo
 
+# Upgrade pyopenssl
+RUN pip3 install pyopenssl --upgrade
+
```

Comment:
From the looks of the error it seems the pip dependency for docker-compose below needs a specific version of pyopenssl. It might be easier to find the version you've got working now and then pin it below on line 110

Reply:
Not sure it's easier compared to doing nothing.
Why the "upgrade" is not good solution. Anyway, I will remove `docker-compose` at some point since we've already switched to the golang version in LmP.

Reply:
It just takes ages to verify whether a new dockerfile works or not due to the bad internet connection at the place where I am currently.
The version should be `pyOpenSSL==23.0.0`. I'll check it out once I get better connection.